src/plugins/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -72,6 +72,16 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
   const addDataFeatures = getFeaturesByCategory(FeatureCatalogueCategory.DATA);
   const manageDataFeatures = getFeaturesByCategory(FeatureCatalogueCategory.ADMIN);
   const devTools = findFeatureById('console');
+  const noDataConfig: KibanaPageTemplateProps['noDataConfig'] = {
+    solution: 'Analytics',
+    logo: 'logoKibana',
+    actions: {
+      beats: {
+        href: addBasePath(`home#/tutorial_directory`),
+      },
+    },
+    docsLink: 'https://www.elastic.co/kibana/',
```

Comment:
I know it's an extra step, but could we add this product page URL to the [core docLinks service](https://github.com/elastic/kibana/blob/master/src/core/public/doc_links/doc_links_service.ts) and import it from there instead?

We've done something similar for elastic.co/maps: 
https://github.com/elastic/kibana/blob/8088565ee9b4f7efe6ea916d1e36b9c7f96bcf3a/src/core/public/doc_links/doc_links_service.ts#L267

Reply:
Switched to a docs link (instead of the previous product link). This also meant returning the `KIBANA_DOCS` url since it points to the current version of the docs versus the other kibana links value that drops you on the full list of versions, which seems not very helpful.

Reply:
> Switched to a docs link (instead of the previous product link)

Ah, so if you want to use the docs link instead, then I don't think you'll need to change anything in the docLinks service itself, as this is already exposed under `links.kibana`: 
https://github.com/elastic/kibana/blob/fa99b8e4d18e1e96e85cf40954490ed308ebb9f0/src/core/public/doc_links/doc_links_service.ts#L147

I think you would only need to update the service if you wanted to use the product link.

Reply:
In the PR summary, it states that we want the `learn more` link to go to https://www.elastic.co/kibana/, so it's now using `${ELASTIC_WEBSITE_URL}kibana`.

Reply:
This was my original guess not seeing guidance from Kibana Product, and then I thought docs might be better and aligned with what Obs is doing. @alexfrancoeur which of these do you prefer for the 'learn more' link?

1. https://www.elastic.co/kibana/ or
2. https://elastic.co/guide/en/kibana/{CURRENT_VERSION}/

Apologies if I missed this being answered elsewhere.

Reply:
Discussed with Alex and we'd like to align with what other solutions are doing as we also feel the docs information is a better fit at this point in the experience. Security and Observability are also pointing the guide/docs.